assets/src/design-system/components/menu/menu.js

```diff
@@ -96,7 +106,7 @@ const Menu = ({
 
     highlighedOptionEl.focus();
     listEl.scrollTo(0, highlighedOptionEl.offsetTop - listEl.clientHeight / 2);
-  }, [focusedIndex]);
+  }, [focusedIndex, isMenuFocused, listRef]);
```

Comment:
I think we don't need `listRef` here. `listRef` will always be referentially equal so `useEffect` won't trigger off of it

Reply:
I think a linting rule `exhaustive-deps` is at work here

Reply:
the linting rule is indeed at work here. I can remove it from the list or leave it.

Reply:
Normally I'd remove it from the dependency array but if the lint rule enforces this then 🤷. There's no harm that comes from listing that as a dependency